### DISCUSSION TITLE
Showcase @airsdk.dev ?

### DISCUSSION THREAD

**Discussion by 2jfw** [airsdk/Adobe-Runtime-Support#1158] [Ideas]
Maybe it is planed already but I would think that a **Showcase Section** on **https://airsdk.dev** would be really great.
Separated to **Games** and **Applications** (and maybe platforms sort options) developers could showcase their work.

In the Starling forum (https://forum.starling-framework.org/) there have been some great and amazing games be shown and what better place would there be than on that new AIR SDK page?

Developers of successful Steam Games could also be contacted and asked if there Games could be added to that page, so people can see, that AIR is actually _alive_!

**A list of Games here:** (sort them be Release Date or Rating)
https://steamdb.info/tech/Engine/Adobe_AIR/

**Some of the recent highlights from Starling Forum:**

**Guardians of Elderon:**
https://forum.starling-framework.org/d/23356-my-first-game-guardians-of-elderon-beta/19

**Bioocean Fantasy:**
https://apps.apple.com/au/app/bioocean-fantasy/id864800888

**BrokenRanks:**
https://www.brokenranks.com/

I could imagine sharing some public information and showcase the project we are working on - one of the biggest Video/Surveillance Security Management Systems in the World (Hemisphere SeMSy 5), which FrontEnd UI is also done in AIR (with Flex). SeMSy 5 (and also the predecessor SeMSy 3, a true old school Flash App wrapped with Zinc) is mostly used in Macaus biggest Casinos with 15k+ (up to 45k+)  Cameras etc... but it is much more than just video surveillance, also used for city surveillance, airports, sports stadios, you name it
Just a have a look at dallmeier.com:

- https://www.dallmeier.com/solutions
- https://www.dallmeier.com/technology/software

I could fill pages now but will likely postpone this to a later date here

**Comment by Elintondm** [comment]
There are some showcases here:

https://airnativeextensions.com/showcase

Em qua., 15 de set. de 2021 às 11:24, Jan ***@***.***>
escreveu:

**Reply by 2jfw** [reply]
Starling Website does also have some Showcases:

https://gamua.com/starling/games/

[**EDIT:** @PrimaryFeather which haven't been maintained for a while I guess... *hint* ;)]

**Reply by PrimaryFeather** [reply]
🙊 You're right — I should definitely add some more recent games into that list!! 😬

**Comment by GeneralVimes** [comment]
This is definitely what community needs!
There are several showcase threads on r/as3:

https://www.reddit.com/r/as3/comments/p6mzwc/madewithair_july2021/
https://www.reddit.com/r/as3/comments/p6mypi/madewithair_august2021/
https://www.reddit.com/r/as3/comments/p698rt/made_with_air_showcase_august_2021/
https://www.reddit.com/r/as3/comments/lmpkn1/share_the_current_progress_of_your_as3_game_or_app/

**Comment by PrimaryFeather** [comment]
The Starling Wiki also contains a showcase where developers can add their games on their own. The information it provides is very basic, but the list is really extensive! 

https://wiki.starling-framework.org/games/start

**Reply by 2jfw** [reply]
Oh, okay..  never came across that page - maybe you should cross-link both showcases at the top of the page?

**Reply by PrimaryFeather** [reply]
If you scroll down on the main (Gamua) showcase, there is a button "even more games" that forwards to the wiki page. But I'm sure you're not the only one who has never seen that list. 😜 

"top of the page" is probably the key — I'll change that.

In any case, I am all on your side, an official list directly on Harman's page would be great to raise awareness and trust in the platform!

**Comment by marchbold** [comment]
We initially had been focussing on docs / news / tutorials / getting started type information on airsdk.dev but I see no reason we couldn't run a showcase there eventually. 

I think the main focus initially should be updating the resources and presenting a modern professional front to the documentation and developer guides/tutorials. A lot has changed since AIR 32 but we still don't have any updated docs so this is where we are concentrating efforts. Too many links lead to adobe where they point to harman or mention "no longer maintained" / "archived" etc...

**Comment by marchbold** [comment]
Showcase and community sections are now live on airsdk.dev:

- [Showcase](https://airsdk.dev/showcase)
- [Community](https://airsdk.dev/community)

If you want to put an application up there you can follow the guide to do a PR or just shoot me the details on your app along with an image for the site.
